firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionViewController.swift

```diff
@@ -334,6 +334,11 @@ class TrackingProtectionViewController: UIViewController, Themeable, Notifiable,
         updateViewDetails()
         updateProtectionViewStatus()
         applyTheme()
+        getCertificates(for: viewModel.url) { certificates in
```

Comment:
We probably want a `weak self` here if this call could run long to avoid retaining the viewController.

Reply:
why does the tracking protection VC need to know about this? to better understand, could this be done in the initialization of the certificate view?

Reply:
@ih-codes yes, [weak self] may be better there
@cyndichin may you be more exactly, please?

Reply:
I would like to understand why we would like to initialize `getCertificates` in the `TrackingViewController`. I could be misinterpreting this, so wanted to understand the reasoning behind adding the code here.

Reply:
@cyndichin just want to have the certificates loaded for the Certificates screen also for the details screen. Don't want to load them, for two separate times.

Reply:
okay, we may need to do this for the error pages work as well. just wanted to make sure we're not duplicating efforts if possible. thanks!

Reply:
I'll let @ih-codes review this PR more thoroughly + lean on her approval since your both doing a great job at it, feel free to ping me if you need extra eyes.